ambry-utils/src/main/java/com.github.ambry.utils/Utils.java

```diff
@@ -180,10 +183,8 @@ public static ByteBuffer readShortBuffer(DataInputStream input) throws IOExcepti
   public static Thread newThread(Runnable runnable, boolean daemon) {
     Thread thread = new Thread(runnable);
     thread.setDaemon(daemon);
-    thread.setUncaughtExceptionHandler(new Thread.UncaughtExceptionHandler() {
-      public void uncaughtException(Thread t, Throwable e) {
-        //error("Uncaught exception in thread '" + t.getName + "':", e)
-      }
+    thread.setUncaughtExceptionHandler((t, e) -> {
```

Comment:
Yeah, this is better. I am assuming all those callers use this method to create new threads.